arviz/tests/base_tests/test_data.py

```diff
@@ -77,6 +79,18 @@ def no_remote_data(monkeypatch, tmpdir):
     )
 
 
+@pytest.fixture
+def data_random():
+    data = np.random.normal(size=(4, 500, 8))
+    idata = from_dict(
+        posterior={"a": data[..., 0], "b": data},
+        sample_stats={"a": data[..., 0], "b": data},
+        observed_data={"b": data[0, 0, :]},
+        posterior_predictive={"a": data[..., 0], "b": data},
+    )
+    return idata
```

Comment:
In views of `extend` method which will require multiple inference data objects, I propose the following pattern used in https://github.com/arviz-devs/arviz/blob/master/arviz/tests/helpers.py#L43:

```
def create_data_random(groups=None, seed=10):
    if groups is None:
        groups = ["posterior", "sample_stats", "observed_data", "posterior_predictive"]
    rng = np.random.default_rng(seed)
    data = rng.normal(size=(4, 500, 8))
    idata_dict = dict(
        posterior={"a": data[..., 0], "b": data},
        sample_stats={"a": data[..., 0], "b": data},
        observed_data={"b": data[0, 0, :]},
        posterior_predictive={"a": data[..., 0], "b": data},
        prior={"a": data[..., 0], "b": data},
        prior_predictive={"a": data[..., 0], "b": data},
    )
    idata = from_dict(**{group: ary for group, ary in idata_dict.items() if group in groups})
    return idata

@pytest.fixture(scope="module")
def data_random():
    idata = create_data_random()
    return idata
```

Reply:
Yeah, good idea!